src/components/new-layout/QLayoutDrawer.js

```diff
@@ -20,12 +20,14 @@ export default {
     breakpoint: {
       type: Number,
       default: 992
+    },
+    view: {
+      type: String,
+      validator: v => ['desktop', 'mobile'].includes(v)
```

Comment:
Let's call it "behavior" better. It can take values: "default", "mobile", "desktop". Set its default to "default". Thanks!

Reply:
Done